src/components/HomePage/Banner.js

```diff
@@ -40,7 +42,11 @@ const useStyles = makeStyles(theme => ({
 	timer: {
 		color: 'white',
 		fontWeight: 600,
-		fontSize: '4em'
+		fontSize: '4em',
+		textAlign: 'center',
```

Comment:
The `textAlign` change doesn't appear necessary here, and I suspect that the `textAlign` change above might not be necessary either. I think the `alignItems` fix below might have rendered these unnecessary.